installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -285,6 +285,10 @@ spec:
           env:
             - name: AWX_SKIP_MIGRATIONS
               value: "1"
+            - name: MY_POD_UID
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.uid
```

Comment:
Originally I was going to generate them in python but, as @shanemcd pointed out, you can actually use the one k8s generates. This has the benefit of being reachable to any container in the pod.